README.md

```diff
@@ -46,6 +46,12 @@ commercetools sync is a Java library that could be used to synchronise CTP data
 (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)).
 
 
+Currently this library supports synchronising
+ - [Categories](/docs/usage/CATEGORY_SYNC.md)
+ - [Products](/docs/usage/PRODUCT_SYNC.md) (_Beta_: Not recommended for production use yet.)
+ - [InventoryEntries](/docs/usage/INVENTORY_SYNC.md) (_Beta_: Not recommended for production use yet.)
+ - [ProductTypes](/docs/usage/PRODUCT_TYPE_SYNC.md) (_Beta_: Not recommended for production use yet.)
+ - [Types](/docs/usage/TYPE_SYNC.md) (_Beta_: Not recommended for production use yet.)
```

Comment:
This whole part should be removed. Since we removed the beta flags in the master. And only the new `Types` line should be added above here: https://github.com/commercetools/commercetools-sync-java/pull/333/files#diff-04c6e90faac2675aa89e2176d2eec7d8R18

Reply:
Ready for review: https://github.com/commercetools/commercetools-sync-java/pull/333/commits/a787872ed9c9c5d27f6b0770225f7f3f968264de